src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -99,17 +99,21 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         client.setOAuth2Token(token);
 
         Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
         String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
         List<String> instanceGithubTeams = instance.getSpec().getGithub().getTeams();
         Boolean isAdmin = false;
 
         TeamService teamService = new TeamService(client);
         List<Team> teams = teamService.getTeams(instanceGithubOrg);
 
-        for (Team team : teams) {
-            for (String teamName : instanceGithubTeams) {
-                if (teamName.equals(team.getName())) {
-                    isAdmin = teamService.isMember(team.getId(), new UserService(client).getUser().getLogin());
+        for (Team orgTeam : teams) {
+            for (String kabaneroAdminTeam : instanceGithubTeams) {
+                if (kabaneroAdminTeam.equals(orgTeam.getName())) {
+                    isAdmin = teamService.isMember(orgTeam.getId(), new UserService(client).getUser().getLogin());
```

Comment:
so this won't work if the user is in 1 of 2 admin teams. The loop will go through the first team the user is in and go `true`, but the second loop will turn isAdmin false and return false.

Once the isAdmin flag turns true we need to return or break out of both loops